examples/Vanilla/OnLoad/index.html

```diff
@@ -13,7 +13,18 @@ <h1>OnLoad</h1>
     <button autofocus onclick="window.location.href='../Conversion/index.html'">
       Next
     </button>
-    <script src="https://sdk.real-user-data.eggplant.cloud/latest/rci.min.js"></script>
-    <script type="module" src="./main.js"></script>
+    <script src="./../../../dist/rci-legacy.min.js"></script>
```

Comment:
@robertdumitrescu INFO - All of the examples now use the legacy version - is this intended?

Reply:
Well yes, since those will work everywhere.